test/user.test.js

```diff
@@ -1763,6 +1763,367 @@ describe('User', function() {
     });
   });
 
+  describe('Email Update', function() {
+    describe('User changing email property', function() {
+      var usersId;
+      var currentEmailCredentials = { email: 'm@c.com', password: 'bar' };
+      var updatedEmailCredentials = { email: 'n@d.com', password: 'bar' };
+
+      beforeEach(function(done) {
+        async.series([
+          function(next) {
+            User.create({ email: currentEmailCredentials.email,
+            password: currentEmailCredentials.password }, function(err, user) {
```

Comment:
Wrong formatting/indentation, see https://github.com/strongloop/loopback-contributor-docs/blob/master/style-guide.md#one-arg-per-line

Please apply this and the comment above to the rest of the newly added code in this file.